api/src/main/java/com/nicknathanjustin/streamercontracts/twitch/TwitchServiceImpl.java

```
@@ -71,6 +76,27 @@ public TwitchUser getTwitchUserFromTwitchUserId(@NonNull final String twitchUser
         return TwitchUser.createTwitchUser(details);
     }
 
+    @Override
+    public ResponseEntity sendExtensionRefresh(@NonNull final ContractState contractState, @NonNull final String channelId) {
+        final String url = "https://api.twitch.tv/extensions/message/" + channelId;
+        final String extensionJwt = getTwitchExtensionJwtToken(channelId);
+        final HttpHeaders headers = new HttpHeaders();
+        headers.set("Authorization", "Bearer " + extensionJwt);
+        headers.set("Client-Id", extensionClientId);
+        headers.setContentType(MediaType.APPLICATION_JSON);
+
+        try {
+            final ExtensionMessageRefreshRequest body = new ExtensionMessageRefreshRequest(contractState);
+            final HttpEntity<?> entity = new HttpEntity<>(body, headers);
+            final RestTemplate restTemplate = new RestTemplate();
+
+            return restTemplate.exchange(url, HttpMethod.POST, entity, ResponseEntity.class);
+        } catch (IOException e) {
+            log.error("Could not serialize the contract state.");
+            throw new RuntimeException(e);
```

Why not just re-throw the original exception?